model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -30,6 +30,7 @@ import io.kotest.matchers.should
 import java.io.File
 
 import org.ossreviewtoolkit.model.config.REFERENCE_CONFIG_FILENAME
+import org.ossreviewtoolkit.utils.ort.ORT_PACKAGE_CONFIGURATION_FILENAME
```

Comment:
Should this constant use plural form, like the cli option `--package-curations-file` ?

Reply:
You're right, my bad.

Reply:
While the example file might only contain a single package configuration the `EvaluatorCommand` option `--package-configuration-file` actually reads a list of package configurations from this file (and should therefore be renamed to `--package-configurations-file`). To me it doesn't make sense to introduce a global constant that relates only to this example file. If we introduce the constant it should rather reflect the CLI use case and use plural for the constant name and file name.
Also, note that the other constant `ORT_PACKAGE_CONFIGURATIONS_DIRNAME` also uses plural. And for consistency with package curations `--package-configuration-dir` should also be renamed to `--package-configurations-dir`.